person_signup.md

```diff
@@ -47,8 +47,8 @@ A list of fields specific for POSTing via the Person Signup Helper.
 
 |Name          |Type      |Description
 |-----------    |-----------|--------------
-|add_tags      |strings[]     |An array of tag names corresponding to previously created tags to add to this person when it is created.
-|add_lists     |strings[]     |An array of list names corresponding to previously created lists to add to this person when it is created.
+|add_tags      |strings[]     |An array of tag IDs corresponding to previously created tags to add to this person when it is created.
+|add_lists     |strings[]     |An array of list IDs corresponding to previously created lists to add to this person when it is created.
```

Comment:
I'm all for it basically but note that it'd be a breaking change.  Also we should propagate the change to record canvass helper.  To avoid the breaking change, we could use add_tags_ids and add_lists_ids alongside the old add_tags, add_lists properties.